README.rst

```diff
@@ -82,10 +82,16 @@ then run pyup from the cli::
 Custom Gitlab instance and GitHub Enterprise support
 ----------------------------------------------------
 
-Pyup offer support for custom Gitlab instances and GitHub Enterprise via the provider_url option :
+Pyup offer support for custom Gitlab instances and GitHub Enterprise via the provider_url option::
 
     $ pyup --provider github --provider_url https://github.enterprise/api/v3 --repo=username/repo --user-token=<YOUR_TOKEN>
     $ pyup --provider gitlab --provider_url https://your.gitlab/ --repo=username/repo --user-token=<YOUR_TOKEN>
 
     # The alternative method to add a custom gitlab instance is still valid :
     $ pyup --provider gitlab --repo=username/repo --user-token=<YOUR_TOKEN>@https://your.gitlab/
+
+
+Disable verification of SSL certificate::
+
+    $ pyup --provider github --provider_url https://github.enterprise/api/v3 --repo=username/repo --user-token=<YOUR_TOKEN> --ignore_ssl
```

Comment:
Nice documentation. I was also trying to figure out what was the use case for ignoring SSL certificate. I guess this makes things clear.